.gitignore

```diff
@@ -35,3 +35,7 @@ npm-debug.log
 testem.log
 /.chrome
 /.git
+
+# schematics
+/schematics/**/*.js
+/schematics/**/*.map
```

Comment:
Any reason not to put this under `src`?

Reply:
Would you like to move all the code under src?

Reply:
Nah, just curious; we can move them if we ever need to